src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -1103,6 +1104,101 @@ object Helpers extends SparkSessionWrapper {
     when(url.endsWith("/"), url.substr(lit(0), length(url) - 1)).otherwise(url)
   }
 
+
+  /**
+   * Function separates the raw api response from the enriched api response which contains the meta info of the api call and
+   * returns a dataframe which contains only raw api response.
+   * @param dataFrame
+   * @return
+   */
+  def deriveRawApiResponseDF(dataFrame: DataFrame): DataFrame = {
```

Comment:
There's enough API-specific helpers that I think we could have an APIHelpers module on its own

Reply:
yes i agree